File: SAPS.Api/Dtos/AuthResponse.cs
using SAPS.Api.Models.Generated;

namespace SAPS.Api.Dtos
{
    public class AuthResponse
    {
        public string AccessToken { get; set; } = null!;
        public UserResponseDto User { get; set; } = null!;
        public DateTime ExpiresAt { get; set; }
    }
}

File: SAPS.Mobile/app/src/main/java/vn/edu/fpt/sapsmobile/dtos/AuthResponse.java
package vn.edu.fpt.sapsmobile.dtos;

import vn.edu.fpt.sapsmobile.models.User;

// Models/AuthResponse.java
public class AuthResponse {
    private String accessToken;
    private String refreshToken;
    private User user;
    private String expiresAt;

    // Getters and setters
    public String getAccessToken() { return accessToken; }
    public void setAccessToken(String accessToken) { this.accessToken = accessToken; }
    
    public String getRefreshToken() { return refreshToken; }
    public void setRefreshToken(String refreshToken) { this.refreshToken = refreshToken; }
    
    public User getUser() { return user; }
    public void setUser(User user) { this.user = user; }
    
    public String getExpiresAt() { return expiresAt; }
    public void setExpiresAt(String expiresAt) { this.expiresAt = expiresAt; }
}
